sauron/README.md

```diff
@@ -23,3 +23,14 @@ Here is a fully reptilian example running against [blockstream.info](https://blo
 ```
 lightningd --mainnet --disable-plugin bcli --plugin $PWD/sauron.py --sauron-api-endpoint https://blockstream.info/api/
 ```
+
+You can use also proxy your requests through [Tor](https://www.torproject.org/) by adding
+the `--sauron-use-tor` startup option.
+
+Hidden services are also supported :
+```
+lightningd --testnet --disable-plugin bcli --plugin $PWD/sauron.py --sauron-api-endpoint http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/testnet/api/ --sauron-use-tor
+```
+
+*Just to make it clear, this option doesn't make you automagically anonymous : if you are
+using a public endpoint then you are giving away all links between your coins.*
```

Comment:
Should I drop this ? As I often see a lot of confusion about the effects of using Tor I thought it would be nice to have..

Reply:
The only really public information should be transactions we send out anyway, shouldn't it? Otherwise we process full blocks, grabbing owned outputs without revealing ownership. Behavior can leak that we're a LN node, due to short channel ID lookups, but that's not a huge leak imho.

Reply:
There are also the `getutxout` calls we do with our coins iirc ?

Reply:
Only on `dev-rescan-outputs` iirc, which users usually don't do.